packages/babylon/src/tokenizer/index.js

```diff
@@ -354,6 +354,12 @@ export default class Tokenizer extends LocationParser {
         case charCodes.slash:
           switch (this.input.charCodeAt(this.state.pos + 1)) {
             case charCodes.asterisk:
+              if (this.plugins.flow && this.skipFlowComment()) {
+                this.state.pos += this.skipFlowComment();
+                this.state.hasFlowComment = true;
+                break;
+              }
```

Comment:
Instead of modifying this file, can we move the flow comment parsing logic to `plugins/flow.js`? (You can override the `skipBlockComment` method).

Reply:
Done